config/services.yaml

```diff
@@ -74,7 +75,8 @@ services:
     Keboola\JobQueueInternalClient\Client:
         arguments:
             $internalQueueApiUrl: '%env(JOB_QUEUE_URL)%'
-            $internalQueueToken: '%env(JOB_QUEUE_TOKEN)%'
+            $internalQueueToken: ~
+            $storageApiToken: '%env(STORAGE_API_TOKEN)%'
```

Comment:
This is the actual change - STORAGE_API_TOKEN env is added in https://github.com/keboola/job-queue-daemon/pull/322/files

Reply:
sorry, does `~` mean null or emtpy string?  I don't know the syntax

Reply:
yes https://js.educative.io/answers/how-to-represent-null-values-in-yaml

Reply:
cool, thanks 🍻